ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -369,8 +369,19 @@ TFLiteInterpreter::invoke (const GstTensorMemory *input, GstTensorMemory *output
   if (is_xnnpack_delegated) {
     for (unsigned int i = 0; i < inputTensorMeta.num_tensors; ++i) {
       tensor_ptr = inputTensorPtr[i];
-      g_assert (tensor_ptr->bytes == input[i].size);
-      memcpy (tensor_ptr->data.raw, input[i].data, input[i].size);
+
+      /* Replace g_assert with explicit error check for release builds */
+      if (tensor_ptr->bytes != input[i].size) {
+        ml_loge ("Tensor size mismatch for input tensor %u: expected %zu, got %zu",
+            i, tensor_ptr->bytes, input[i].size);
+        return -EINVAL;
+      }
+
+      /* Check memcpy result to handle potential memory copy failures */
+      if (memcpy (tensor_ptr->data.raw, input[i].data, input[i].size) == NULL) {
+        ml_loge ("Memory copy failed for input tensor %u", i);
+        return -ENOMEM;
```

Comment:
memcpy does not have an error return.
It returns dest pointer. (tensor_ptr->data.raw)